packages/block-editor/src/components/spacing-sizes-control/README.md

```diff
@@ -0,0 +1,113 @@
+# Spacing Sizes Control
+
+The SpacingSizesControl component provides a user interface for controlling spacing values in blocks. It supports single, axial, and separated input controls for different spacing configurations.
+
+## Description
+
+The SpacingSizesControl component is a flexible control that allows users to modify spacing values for different sides of a block. It supports three viewing modes:
+
+1. Single: Control one side at a time
+2. Axial: Control horizontal and vertical sides together
+3. Custom: Control each side separately
+
+![Spacing Sizes Control](https://i.postimg.cc/3RkzzfL6/Screenshot-2025-01-10-at-8-07-55-AM.png)
+
+## Usage
+
+```js
+import {
+	InspectorControls,
+	__experimentalSpacingSizesControl as SpacingSizesControl,
+} from '@wordpress/block-editor';
+import { View } from '@wordpress/primitives';
+import { useCustomUnits } from '@wordpress/components';
+
+const DimensionInput = ( { label, onChange, value = '' } ) => {
+	const availableUnits = [ 'px', 'em', 'rem', 'vw', 'vh' ];
+	const units = useCustomUnits( {
+		availableUnits,
+		defaultValues: { px: 100, em: 10, rem: 10, vw: 10, vh: 25 },
+	} );
+
+	const handleOnChange = ( unprocessedValue ) => {
+		onChange( unprocessedValue.all );
+	};
```

Comment:
I think it’d be fine to not have the wrapper and simply pass `onChange` onto `SpacingSizesControl`. At the least, the `unprocessedValue.all` seems like it shouldn’t be there (per my other comment about `all` not being a supported part of the API).

On the other hand, if it seems nice to demonstrate what the received value will consist of it could be something like this:
```suggestion
	const handleOnChange = ( { top, right = '0px', bottom, left = '0px' } ) => {
		onChange( { top, right, bottom, left } );
	};
```